qmetaobject/src/qmetatype.rs

```diff
@@ -144,6 +147,46 @@ fn register_metatype_qobject<T: QObject>() -> i32 {
     }
 }
 
+pub fn register_metatype_qenum<T: QEnum + QMetaType>(
```

Comment:
You should add #[doc(hidden)]  To all these internal things that are used only from the macro.